cmd/bnsd/app/app_performance_test.go

```diff
@@ -0,0 +1,134 @@
+package app
+
+import (
+	"encoding/hex"
+	"io/ioutil"
+	"testing"
+	"time"
+
+	"github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/weavetest"
+	"github.com/iov-one/weave/x/cash"
+	"github.com/iov-one/weave/x/sigs"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/libs/log"
+)
+
+func BenchmarkBNSD(b *testing.B) {
+	var (
+		alice = weavetest.NewKey()
+		benny = weavetest.NewKey()
+		carol = weavetest.NewKey()
+		david = weavetest.NewKey()
+	)
+
+	type dict map[string]interface{}
+	genesis := dict{
```

Comment:
Nice basic config.
If you want to use this for multiple cases, maybe pull it out into a function?
Or maybe not, will read further